srai/loaders/geoparquet_loader.py

```diff
@@ -19,7 +19,7 @@ class GeoparquetLoader:
     and allows for an automatic index setting and additional geometry clipping.
 
     References:
-        [1] https://github.com/opengeospatial/geoparquet
+        1. https://github.com/opengeospatial/geoparquet
```

Comment:
This ensures correct formatting as a list in docs, am I right?

Reply:
Yes, otherwise `mkdocs` renders them as a single line.